newdle/client/src/components/ParticipantTable.js

```diff
@@ -2,10 +2,9 @@ import React, {useState} from 'react';
 import moment from 'moment';
 import {useSelector} from 'react-redux';
 import {Radio, Icon, Label, Table} from 'semantic-ui-react';
-import {CircularProgressbar, buildStyles} from 'react-circular-progressbar';
+import ProgressRing from './AvailabilityRing';
```

Comment:
Let's import it as `AvailabilityRing` (or rename the file and function to `ProgressRing`)